### DISCUSSION TITLE
How can I add reveal-js in demo.os-js.org?

### DISCUSSION THREAD

**Discussion by ghost** [os-js/OS.js#816] [Q&A] [answered]
Hi all.

*How can I add [reveal-js](https://revealjs.com/) in [demo.os-js.org](https://demo.os-js.org/)? Where can I find documentation for creating programs in os.js?*

**Comment by andersevenrud** [comment, marked as answer]
That site is only for demonstration purposes.

You have to install OS.js yourself and create a new application.

All docs here: https://manual.os-js.org/

In this case an iframe application is probably the way to go:
https://manual.os-js.org/tutorial/iframe/

**Reply by ghost** [reply]
Hi andersevenrud, thank you for feedback.
